src/tactic/ui/container/tab_wdg.py

```diff
@@ -252,37 +256,78 @@ def get_onload_js(my):
         if (header) {
             var num = Math.floor((Math.random()*10000)+1); 
             element_name = element_name + num;
-            //title = title + num;
         }
     }
 
 
     var top_id = top.getAttribute("spt_tab_id");
 
+
+    if (element_name.indexOf("/") != -1) {
+        var full_element_name = element_name;
+        var parts = element_name.split("/");
+        element_name = parts[0];
+        var subelement_name = parts[1];
+    }
+    else {
+        var full_element_name = element_name;
+        var subelement_name = "";
+    }
+
```

Comment:
"var" is a weird thing in javascript.  Most languages must declare "global" while it seems all variables in javascript a global and need to be declared "local", which is what var does ... anyway, I will declare it with var outside the block.